core/src/main/java/com/novoda/noplayer/internal/exoplayer/AvailableAdverts.java

```diff
@@ -18,19 +18,16 @@ private AvailableAdverts() {
      * as the exo-player {@link AdPlaybackState} does not allow transitioning from
      * Skipped to Available adverts.
      *
-     * @param currentPositionInMillis The position after which all adverts will transition from Skipped to Available.
-     * @param advertBreaks            The client representation of the adverts, our source of truth.
-     * @param adPlaybackState         The {@link AdPlaybackState} to mutate with the new states.
+     * @param advertBreaks    The client representation of the adverts, our source of truth.
+     * @param adPlaybackState The {@link AdPlaybackState} to mutate with the new states.
      */
-    static void markAllFutureAdvertsAsAvailable(long currentPositionInMillis, List<AdvertBreak> advertBreaks, AdPlaybackState adPlaybackState) {
+    static void markSkippedAdvertsAsAvailable(List<AdvertBreak> advertBreaks, AdPlaybackState adPlaybackState) {
```

Comment:
We no longer make adverts only in the future available, we make all adverts available again. The only way users cannot see an advert again is to watch it all the way to the end so that it is marked as `PLAYED`.